pkg/capabilities/consensus/ocr3/reporting_plugin.go

```diff
@@ -268,43 +267,11 @@ func (r *reportingPlugin) Reports(seqNr uint64, outcome ocr3types.Outcome) ([]oc
 }
 
 func (r *reportingPlugin) ShouldAcceptAttestedReport(ctx context.Context, seqNr uint64, rwi ocr3types.ReportWithInfo[[]byte]) (bool, error) {
-	info := &pbtypes.ReportInfo{}
-	err := proto.Unmarshal(rwi.Info, info)
-	if err != nil {
-		r.lggr.Error("could not unmarshal info")
-		return false, err
-	}
-
-	resp := map[string]any{}
-	if info.ShouldReport {
-		resp["report"] = []byte(rwi.Report)
-	} else {
-		resp["report"] = nil
-	}
-
-	v, err := values.Wrap(resp)
-	if err != nil {
-		r.lggr.Error("could not wrap report", "payload", resp)
-		return false, err
-	}
-
-	r.lggr.Debugw("ShouldAcceptAttestedReport transmitting", "shouldReport", info.ShouldReport, "len", len(rwi.Report))
-	err = r.r.transmitResponse(ctx, &outputs{
-		CapabilityResponse: capabilities.CapabilityResponse{
-			Value: v,
-		},
-		WorkflowExecutionID: info.Id.WorkflowExecutionId,
-	})
-	if err != nil {
-		r.lggr.Errorw("could not transmit response", "error", err, "weid", info.Id.WorkflowExecutionId)
-		return false, err
-	}
-
-	return false, nil
+	return true, nil
 }
 
 func (r *reportingPlugin) ShouldTransmitAcceptedReport(ctx context.Context, seqNr uint64, rwi ocr3types.ReportWithInfo[[]byte]) (bool, error) {
-	return false, nil
+	return true, nil
```

Comment:
Important for contract config - the schedule needs to be set to [N]

Reply:
please add a comment about why this is always true